CHANGELOG.md

```diff
@@ -13,8 +13,243 @@ _Note: Gaps between patch versions are faulty, broken or test releases._
 
 See [CHANGELOG - v4](https://github.com/webdriverio-boneyard/v4/blob/master/CHANGELOG.md).
 
-## v5.0.0 (2018-12-12)
+---
+
+## v5.0.0 (2018-tbd)
 
 #### :boom: Breaking Change
```

Comment:
We might want to explicitly state that packages such as visual-regression-service most likely will not work out of the box with v5.